src/strategies/multichain/index.ts

```diff
@@ -0,0 +1,82 @@
+import {subgraphRequest} from '../../utils';
+import networks from '../../networks.json';
+import {JsonRpcProvider} from "@ethersproject/providers";
+import strategies from '..';
+
+export const author = 'kesarito';
```

Comment:
This should be your GitHub username
```suggestion
export const author = 'kesar';
```